src/Microsoft.DocAsCode.Glob/FileGlob.cs

```diff
@@ -34,9 +34,15 @@ private static IEnumerable<string> GetFilesCore(string cwd, IEnumerable<GlobMatc
 
         private static IEnumerable<string> GetFilesFromSubfolder(string baseDirectory, string cwd, IEnumerable<GlobMatcher> globs, IEnumerable<GlobMatcher> excludeGlobs)
         {
+            var relativePath = GetRelativeFilePath(cwd, baseDirectory);
```

Comment:
@vicancy fixed. Now both `node_modules/` and `node_modules/**` can work, and only `node_modules/` can be used to avoid `PathTooLongException`.